source/scss/320.scss

```diff
@@ -1,106 +1,44 @@
 header {
+
   @include transp-background();
   height: $height_header;
```

Comment:
Change this variable to a height that works, can probably make this a fixed px height, maybe `100px`? See what works

Reply:
done